lib/services/plugins-service.ts

```diff
@@ -88,18 +92,20 @@ export class PluginsService implements IPluginsService {
 				} else {
 					this.$errors.fail("Plugin '%s' is already installed", pluginName);
 				}
+			} else if(installedPlugin && version) {
```

Comment:
also if I have the plugin installed and after that I specify invalid version, the error message is not very clear:
```
~/Work/icenium-cli/scratch/app13 $ appbuilder plugin add toast@2.2.2 // 2.2.2 is not a valid version
Plugin 'toast' is already installed. The currently installed version is '2.0.4'.
```